app/helpers/venues_helper.rb

```diff
@@ -0,0 +1,2 @@
+module VenuesHelper
```

Comment:
Please remove if this unused)

Reply:
I've deleted only this file, but I think, we can remove all folder helper, cause there are no views?